schemas/common/descriptors/search/asset.example.1.json

```diff
@@ -0,0 +1,448 @@
+{
```

Comment:
Is this an attempt to revise the asset schema with the search extensions?  Be aware that asset is undergoing change right now, so this will also probably change...

Reply:
This is an example - I can amend it later if required. We will need to later revise actual asset schema with the search extensions .That could be done later once the change underway is complete.